metrics/metrics_collector.go

```diff
@@ -10,12 +11,20 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+var (
+	normDomain = flag.Float64("normal.domain", 0.0002, "The domain for the normal distribution.")
+	normMean   = flag.Float64("normal.mean", 0.00001, "The mean for the normal distribution.")
+)
```

Comment:
These should probably go in `metrics_cli.go`.  They're CLI flags so the seem a better fit there.

Reply:
@Aneurysm9 thanks. I have updated.